cmd/katalyst-agent/app/options/sysadvisor/qosaware/model/borwein/borwein.go

```diff
@@ -24,21 +24,32 @@ import (
 
 type BorweinOptions struct {
 	InferenceServiceSocketAbsPath string
+	NodeFeatureNames              []string
+	ContainerFeatureNames         []string
 }
 
 func NewBorweinOptions() *BorweinOptions {
-	return &BorweinOptions{}
+	return &BorweinOptions{
+		NodeFeatureNames:      []string{},
+		ContainerFeatureNames: []string{},
+	}
 }
 
 // AddFlags adds flags to the specified FlagSet.
 func (o *BorweinOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.InferenceServiceSocketAbsPath, "borwein-inference-svc-socket-path", o.InferenceServiceSocketAbsPath,
 		"socket path which borwein inference server listens at")
+	fs.StringSliceVar(&o.NodeFeatureNames, "borwein-node-feature-names", o.NodeFeatureNames,
```

Comment:
after discussion, we use config file to configure NodeFeatureNames and ContainerFeatureNames.